pkg/odo/genericclioptions/client.go

```diff
@@ -3,16 +3,24 @@ package genericclioptions
 import (
 	"github.com/openshift/odo/pkg/kclient"
 	"github.com/openshift/odo/pkg/occlient"
-	"github.com/spf13/cobra"
 )
 
 // Client returns an oc client "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"d for this command's options
-func Client(command *cobra.Command) (*occlient.Client, error) {
-	return client(command)
+func Client() (*occlient.Client, error) {
+	ocClient, err := client()
+	if err != nil {
+		return nil, err
+	}
+	kClient, err := kClient()
+	if err != nil {
+		return nil, err
+	}
+	ocClient.SetKubeClient(kClient)
+	return ocClient, nil
 }
 
-// client creates an oc client based on the command flags
-func client(command *cobra.Command) (*occlient.Client, error) {
+// client creates an oc client
+func client() (*occlient.Client, error) {
```

Comment:
It's a little confusing that `client` function creates/returns an `occlient.Client` while for kclient we have a different function whose name represents what it does - `kClient()`.

My concern is that it's going to seem more confusing as time passes. Can we rename it to `ocClient()` instead?

Reply:
Fixed